src/components/MyKiva/BorrowerCarousel.vue

```diff
@@ -0,0 +1,239 @@
+									:default-image="{ width: 80, faceZoom: 50 }"
+									:hash="getBorrowerHash(loan)"
+									:images="[
+										{ width: 80, faceZoom: 50, viewSize: 1024 },
+										{ width: 72, faceZoom: 50, viewSize: 734 },
+										{ width: 64, faceZoom: 50 },
+									]"
+								/>
+							</div>
+							<h5 class="tw-text-center tw-text-ellipsis tw-line-clamp-2 tw-whitespace-normal">
+								{{ getBorrowerName(loan) }}
+							</h5>
+						</div>
+					</KvTab>
+					<KvTab v-if="loans.length > 9">
+						<a
+							href="/portfolio/loans" v-kv-track-event="[
+								'portfolio',
+								'click',
+								'view-all'
+							]"
+						>View all</a>
+					</KvTab>
+				</template>
+				<template #tabPanels>
+					<KvTabPanel v-for="(loan, index) in loans" :key="index" :id="loan.id">
+						<p class="tw-hidden" :id="loan.id"></p>
+					</KvTabPanel>
+				</template>
+			</KvTabs>
+			<div class="carousel-container">
+				<KvCarousel
+					ref="carousel"
+					class="borrower-carousel tw-w-full md:tw-overflow-visible"
+					:multiple-slides-visible="true"
+					:slide-max-width="singleSlideWidth"
+					:embla-options="{ loop: false, align: 'center'}"
+				>
+					<template v-for="(loan, index) in loans" #[`slide${index+1}`] :key="loan.id || index">
+						<BorrowerStatusCard :loan="loan" />
+					</template>
+				</KvCarousel>
+			</div>
+			<div class="tw-text-right" v-if="hasCompletedBorrowers">
+				<a
+					class="tw-text-h5"
+					href="/portfolio/loans" v-kv-track-event="[
+						'portfolio',
+						'click',
+						'see-all-borrowers'
+					]"
+				>See all borrowers</a>
+			</div>
+		</div>
+	</MyKivaContainer>
+</template>
+
+<script setup>
+import KvTabs from '@kiva/kv-components/vue/KvTabs';
+import KvTab from '@kiva/kv-components/vue/KvTab';
+import KvTabPanel from '@kiva/kv-components/vue/KvTabPanel';
+import KvCarousel from '@kiva/kv-components/vue/KvCarousel';
+import KvButton from '@kiva/kv-components/vue/KvButton';
+import BorrowerImage from '#src/components/BorrowerProfile/BorrowerImage';
+import MyKivaContainer from '#src/components/MyKiva/MyKivaContainer';
+import KvLoadingPlaceholder from '@kiva/kv-components/vue/KvLoadingPlaceholder';
+import { isLoanFundraising } from '#src/util/loanUtils';
+import {
+	defineProps,
+	ref,
+	computed,
+	toRefs,
+	inject,
+	onMounted
+} from 'vue';
+import {
+	PAYING_BACK,
+	ENDED,
+	DEFAULTED
+} from '#src/api/fixtures/LoanStatusEnum';
+import BorrowerStatusCard from './BorrowerStatusCard';
+
+const props = defineProps({
+	/**
+   * Array of loans
+   * */
+	loans: {
+		type: Array,
+		default: () => ([]),
+		required: true,
+	},
+	isLoading: {
+		type: Boolean,
+		default: false,
+	},
+});
+
+const $kvTrackEvent = inject('$kvTrackEvent');
+const emit = defineEmits(['selected-loan']);
+
+const { loans } = toRefs(props);
+const carousel = ref(null);
+
+const hasActiveLoans = computed(() => {
+	return loans.value.some(loan => loan?.status === PAYING_BACK || isLoanFundraising(loan));
```

Comment:
This should only be checking the loan status. It should check if the loan is FUNDED, FUNDRAISING, PAYING_BACK, or RAISED.

isLoanFundraising is checking against loan reservations to see if any shares are available for the user to add to their basket, which we don't need for the status cards.